src/Microsoft.OData.Core/UriParser/SemanticAst/ODataPathExtensions.cs

```diff
@@ -21,7 +21,7 @@ namespace Microsoft.OData.UriParser
     /// <remarks>
     /// The values that these methods compute are not cached.
     /// </remarks>
-    internal static class ODataPathExtensions
+    public static class ODataPathExtensions

```

Comment:
Are there particular methods that we need to expose? Can we keep the others `internal`? I ask because most of these methods need to have some quality adjustments made (line 34, for example, will throw `NullReferenceException` if `path` is `null`). Can you please make such adjustments for the methods that need to be exposed, and mark the ones that don't `internal`?

Reply:
#Resolved